src/handlers/dashboard/get_chart_data.py

```diff
@@ -92,13 +97,20 @@ def _format_payload(df: pandas.DataFrame, query_params: dict, filters: list) ->
     payload["totalCount"] = int(df["cnt"].sum())
     if "stratifier" in query_params.keys():
         payload["stratifier"] = query_params["stratifier"]
+        counts = {}
+        for unique_val in df[query_params["column"]]:
+            df_slice = df[df[query_params["column"]] == unique_val]
```

Comment:
I'm a poor man from a poor family - what is this code doing? You grab the column slice - then compare that to a variable and look up a boolean in the dataframe? Or is this some Pandas bullshit where it overrides `==` to return a special object that can be sliced like a lambda by the `[]` operator (and presumably for normal cases just be truthy)?

So this is getting all rows where that column matches the value. Got it. I'm ambivalent on whether this is more or less readable than whatever method-based version Pandas offers to do the same thing - just threw me for a bit.

Reply:
>  Or is this some Pandas bullshit 

Ding ding ding. `df[query_params["column"]] == unique_val` creates a Series of true/false values, and `df[]` around that takes that input and uses it as a mask to select only the rows that are true. I could explictly store this thing in a variable with mask in the name?

Reply:
moved these to mask vars for readability